### PR TITLE
CB-5033 remove reshadow from log viewer and other packages

### PR DESCRIPTION
**Pull request by devnaumov**

### CODE REVIEW

File: webapp/packages/core-utils/src/getProgressPercent.test.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  it('returns 0% when nothing is done', () => {
+    expect(getProgressPercent(0, 100)).toBe(0);
+  });
+
+  it('returns 100% when done equals total', () => {
+    expect(getProgressPercent(100, 100)).toBe(100);
+  });
+
+  it('does not exceed 100%', () => {
+    expect(getProgressPercent(110, 100)).toBe(100);
```
Comment (sergeyteleshev) [inline webapp/packages/core-utils/src/getProgressPercent.test.ts:25]: i dont like that we dont validate anywhere that first argument (done) cause we trust backend here 100% and cannot track a bug once it appeared here
and with current helper implementation we like close the eyes on it and make candy out of its invalid data came with event

File: webapp/packages/core-utils/src/getProgressPercent.test.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  });
+
+  it('does not drop below 0%', () => {
+    expect(getProgressPercent(-10, 100)).toBe(0);
+  });
+
+  it('handles total as zero without throwing error', () => {
+    expect(getProgressPercent(10, 0)).toBe(100);
+  });
```
Comment (sergeyteleshev) [inline webapp/packages/core-utils/src/getProgressPercent.test.ts:34]: also please add test case with NaN value. cause it is also a number type from typescript perspective of view

File: webapp/packages/core-utils/src/getProgressPercent.test.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+});
```
Comment (sergeyteleshev) [inline webapp/packages/core-utils/src/getProgressPercent.test.ts:35]: plus the second argument (total) also can be tested with all of the test cases you created above for the first one (done). cause something can go wrong here eigther one day

File: webapp/packages/plugin-log-viewer/src/LogViewer/LogEntry.tsx [hunk 4]
```diff
@@ -76,19 +40,27 @@ export const LogEntry = observer<Props>(function LogEntry({ item, onSelect, sele
...
   }
 
-  return styled(useStyles(style))(
-    <TableItem item={item.id} className={className} {...use({ selected })}>
-      <TableColumnValue title={item.type} centerContent flex {...use({ icon: true })}>
-        <icon-box>{icon && <IconOrImage icon={icon} />}</icon-box>
+  return (
+    <TableItem item={item.id} className={s(styles, { selected }, className)}>
+      <TableColumnValue className={s(styles, { icon: true })} title={item.type} centerContent flex>
+        <Container>{icon && <IconOrImage className={s(styles, { iconOrImage: true })} icon={icon} />}</Container>
```
Comment (sergeyteleshev) [inline webapp/packages/plugin-log-viewer/src/LogViewer/LogEntry.tsx:46]: feels like container should be with `center` prop

File: webapp/packages/plugin-log-viewer/src/LogViewer/LogEntry.tsx [hunk 5]
```diff
@@ -76,19 +40,27 @@ export const LogEntry = observer<Props>(function LogEntry({ item, onSelect, sele
...
         {displayTime}
       </TableColumnValue>
       <TableColumnValue>
-        <message-cell>
-          <message title={message}>{isError ? <Link onClick={() => onSelect(item)}>{message}</Link> : message}</message>
-        </message-cell>
+        <div className={s(styles, { messageCell: true })}>
+          <div className={s(styles, { message: true })} title={message}>
+            {isError ? (
+              <Link className={s(styles, { icon: true })} onClick={() => onSelect(item)}>
```
Comment (sergeyteleshev) [inline webapp/packages/plugin-log-viewer/src/LogViewer/LogEntry.tsx]: `link` class here

File: webapp/packages/plugin-resource-manager-scripts/src/ResourceManagerScripts.tsx [hunk 6]
```diff
@@ -6,31 +6,21 @@
...
-  }
-`;
-
 export const ResourceManagerScripts: TabContainerPanelComponent = observer(function ResourceManagerScripts() {
   const translate = useTranslate();
 
-  return styled(styles)(
-    <container>
+  return (
+    <Container vertical>
```
Comment (sergeyteleshev) [inline webapp/packages/plugin-resource-manager-scripts/src/ResourceManagerScripts.tsx]: add `overflow` prop here

**Review by sergeyteleshev — CHANGES_REQUESTED** [review]

**Review by Wroud — APPROVED** [review]

**Review by sergeyteleshev — APPROVED** [review]

**Pull request merged**